app/graphql/resolvers/rdf.ts

```diff
@@ -301,7 +301,8 @@ export const dataCubeComponents: NonNullable<
           position: r.position,
         }));
       const values = uniqBy(
-        [...relatedValues, ...rawValues],
+        // rawValues first, because we prefer these
+        [...rawValues, ...relatedValues],
```

Comment:
I do not remember what raw and related are 😅 , and why we would prefer those ? if you have this in mind could you write a small comment with a bit more explanation ?

Reply:
This logic was later changed, at first I thought we can simply include these limit values in `values`, but this turned out to not be the best idea :) I could have cleaned it up beforehand, sorry for that 🫣